Prototyper/Classes/Login/LoginView.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  LoginView.swift
+//  KeychainSwift
+//
+//  Created by Raymond Pinto on 01.12.19.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @EnvironmentObject var model: Model
+    @Environment(\.presentationMode) var presentationMode
+    @State var userid: String = ""
+    @State var password: String = ""
+    @State var showingAlert: Bool = false
```

Comment:
I would suggest to name to boolean flag after the type or alert you want to show.